apps/server/src/modules/user/legacy/controller/admin-api-students.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { Controller, Get, Param, Query } from '@nestjs/common';
+import { EntityNotFoundError, ForbiddenOperationError, ValidationError } from '@shared/common';
+import { Authenticate, CurrentUser, ICurrentUser } from '../../../authentication';
+import { UsersAdminContextEnum } from '../enum';
+import { UserByIdParams, UserListResponse, UserResponse, UsersSearchQueryParams } from './dto';
+import { AdminApiUsersUc } from '../uc';
+
+@ApiTags('AdminStudents')
+@Authenticate('jwt')
+@Controller('users/admin/students')
+export class AdminApiStudentsController {
+	constructor(private readonly uc: AdminApiUsersUc) {}
+
+	@Get()
+	@ApiOperation({
+		summary: 'Returns all students which satisfies the given criteria.',
+	})
+	@ApiResponse({ status: 200, type: UserListResponse, description: 'Returns a paged list of students.' })
+	@ApiResponse({ status: 400, type: ValidationError, description: 'Request data has invalid format.' })
+	@ApiResponse({ status: 401, type: ForbiddenOperationError, description: 'Not authenticated.' })
```

Comment:
Status and error do not correspond here correctly. `ForbiddenOperationError` has status 403.